### PR TITLE
docs: Update InputFieldWrapper docs

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Adding the docs for InputFieldWrapper, it's mobile only.

### Added

- Added the stories for InputFieldWrapper

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# Input Field Wrapper (Internal)
+
+This component is used to wrap input fields such as `InputText` and
+`InputPressable` to provide common patterns and functionality present in our
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:8]: Can we link to these components?

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+input designs, such as: suffix/prefix icons, placeholders, etc.
+
+Note: example does not have a child component, so no interaction will occur on
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:11]: I don't think you need this here anymore.  The example is in the Mobile folder and it has the disclaimer on it.

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+[Consistency and Standards](https://www.nngroup.com/articles/ten-usability-heuristics/).
+With clearer guidance around the purpose of inputs, the user is able to better
+focus on the task at hand. See
+[InputFieldWrapper/Prefix and Suffix example](../?path=/story/components-private-inputfieldwrapper-mobile--prefix-and-suffix).
+
+### Clearable
+
+When setting `showClearAction` to `true` you must also provide an `onClear`
+callback that will clear the input you are wrapping. See `InputDateTime` and
+`InputPressable` or `InputText` for examples.
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:35]: Can we also link to these? And maybe take them out of the backticks

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# Input Field Wrapper (Internal)
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:5]: I would consider removing `(Internal)` now that it's in a Private section, this is implied

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+This component is used to wrap input fields such as `InputText` and
+`InputPressable` to provide common patterns and functionality present in our
+input designs, such as: suffix/prefix icons, placeholders, etc.
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:9]:
```suggestion
input designs, such as suffix/prefix icons, placeholders, etc.
```

**Review by taylorvnoj — CHANGES_REQUESTED** [review]
Looks good just a few cleanups on the docs page

File: docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# InputFieldWrapper
+
+This component is used to wrap input fields such as
+[InputText](../?path=/docs/components-forms-and-inputs-inputtext-docs--page) and
+[InputPressable](../docs/components-forms-and-inputs-inputpressable-docs--page)
```
Comment (taylorvnoj) [inline docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx:9]: Last thing - I think this InputPressable and the one on line 35 are missing the path part, they're broken

Comment (rodrigoeidelvein) [reply docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx]: Oh god, I copied the wrong part, thanks for catching it

**Review by taylorvnoj — APPROVED** [review]
Thanks for moving so fast on these last few Rodrigo 🏃‍♂️

**Pull request merged**